devops/ansible/nginx_install.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install nginx server 
+  hosts: stagingserver
+  remote_user: wimops
+  roles:
+    - nginx
+...
```

Comment:
Is this the main playbook? It doesn't look complete. Which version of PHP is it running? (It needs to run PHP 7). What about mysql? Also, the server needs to install an SSL certificate so it's accessible over HTTPS.

This is my fault. The user story Acceptance Criteria should have been more explicit.

The goal for this story is to have a staging server that is functionally equivalent to the server produced by this manual process:

- https://kb-smc.chapman.edu/?p=2086

Let's call the main playbook `provision.yml`.